wavepool/models.py

```diff
@@ -32,7 +32,10 @@ def teaser(self):
 
     @property
     def source_divesite_name(self):
-        return 'Industry Dive'
+        # Implementation based on naive assumption that URL is in a
+        # <protocol>://www.<hostname>.xyz/asdf-asdf-asdf format
+        hostname = self.source.split('/')[2].split('.')[1]
+        return DIVESITE_SOURCE_NAMES[hostname]
```

Comment:
`source_divesite_name` was hardcoded before. I have modified this to resolve the hostname from the URL and then return the lookup from `DIVESITE_SOURCE_NAMES`

Reply:
i really like and prefer an implementation similar to ref[https://github.com/industrydive/wavepool/pull/2/files#diff-c771b7f4c489c8c480a8d33826b711772255510cf4595ece8e761a395109d81aR35] 

```py
    def source_divesite_name(self):
        return 'Industry Dive'
        """ Return the real divesite source name for the newspost's source
        """
        source_dive = 'Industry Dive'
        for dive_source in DIVESITE_SOURCE_NAMES:
            url_parts = self.source.split('/')
            dive_domain = url_parts[2]
            domain_parts = dive_domain.split('.')
            dive_domain = domain_parts[1]
            if dive_domain == dive_source:
                return DIVESITE_SOURCE_NAMES[dive_domain]
        return source_dive
```